### DISCUSSION TITLE
Частота дискретизации синтезированного голоса

### DISCUSSION THREAD

**Discussion by Lukina-Tver** [wladradchenko/wunjo.wladradchenko.ru#25] [Q&A] [answered]
Подскажите, есть ли возможность повысить частоту дискретизации получаемого голоса хотя бы до 44,1 кГц? В наших экспериментах с голосом получались файлы с частотой 22,05 кГц, а один даже оказался 16 кГц. У меня неплохой музыкальный слух, и, по моему мнению, голоса с низкой частотой дискретизации звучат глуховато и неестественно.

**Comment by wladradchenko** [comment, marked as answer]
Привет. Нет, не получится, так как данные на которых обучались модели 22.05 кГц или 16 кГц, искусственное повышение дискретизации качество результата не улучшит
